src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,86 @@
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { Loader } from '../Loader';
+import { PersonInfo } from '../PersonInfo';
+
+const findParents = (people: Person[]) => (
```

Comment:
```suggestion
const getPreparedPeople = (people: Person[]) => (
```

Also, I would move it to separate file as a helper function